src/main/resources/api.yml

```diff
@@ -0,0 +1,503 @@
+info:
+  title: LostInBerlin API
+  description: Move your app forward with the LostInBerlin API
+  version: "1.0.0"
+# the domain of the service
+host: lost-in-berlin.com
+# array of all schemes that your API supports
+schemes:
+- https
+# will be prefixed to all paths
+basePath: /api/v1
+securityDefinitions:
+  apikey:
+    type: token
+    name: server_token
+    in: header
+produces:
+- application/json
+paths:
+  /business:
+    get:
```

Comment:
since this url refers to a collection of businesses, it is more correct to have it in plural: `businesses`. Below you have 'orders' for the context of orders.

Reply:
done